pilot/helpers/cli.py

```diff
@@ -252,12 +258,15 @@ def execute_command_and_check_cli_response(command, timeout, convo):
 
     Returns:
         tuple: A tuple containing the CLI response and the agent's response.
+            - cli_response (str): The command output.
+            - llm_response (str): 'DONE' or 'NEEDS_DEBUGGING'
     """
-    cli_response, response = execute_command(convo.agent.project, command, timeout)
-    if response is None:
-        response = convo.send_message('dev_ops/ran_command.prompt',
+    # TODO: Prompt mentions `command` could be `INSTALLED` or `NOT_INSTALLED`, where is this handled?
```

Comment:
for devops agent while installing tech dependencies response will be `INSTALLED` or `NOT_INSTALLED` but for developer agent, or in other words commands executed while coding eg. npm run start, it can return `DONE` or `NEEDS_DEBUGGING`.